package-lock.json

```diff
@@ -0,0 +1,5261 @@
+{
```

Comment:
Is there a reason you're including this? We've not included a package-lock.json file before.

Reply:
package-lock.json is [recommended](https://docs.npmjs.com/files/package-lock.json) to be maintained in source control. Benefits:
* Determinism. The same package version will be restored for all users, regardless of updates on npm, until this file is updated.
* Time Travel. Moving to an earlier commit will cause `npm i` to restore packages as they existed at the time of that commit. 

I looked through user complaints, and there are two classes:
* LIBRARIES can get into trouble since they don't have control over their dependencies as resolved in target apps. Folks believe that by sacrificing build-time determinism they increase the likelyhood of finding bugs in dependency packages. We ship an app, so this does not apply. In fact, the determinism increases the likelyhood that what we ship is what we tested.
* Noisy diffs. Time will tell...